pyproject.toml

```diff
@@ -19,18 +19,31 @@ nxontology_data = "nxontology_data.commands:cli"
 python = "^3.10"
 bioregistry = "^0.8.3"
 bioversions = "^0.5.98"
-fire = "^0.4.0"
+fire = "^0.5.0"
 jupyter = "^1.0.0"
 nbconvert = "^6.2.0"
 notebook = "^6.4.4"
 nxontology = "^0.5.0"
 openpyxl = "^3.0.9"
-pandas = "^1.4.1"
+pandas = "^2.0.3"
 papermill = "^2.3.3"
 requests = "^2.26.0"
 rdflib = "^6.2.0"
 fsspec = "^2022.3.0"
 numpy = "^1.22.3"
+torch = [
+    { version = "^2.0.1", markers = "sys_platform != 'linux'", source = "pypi" },
+    # Use different src for linux, see: https://pytorch.org/get-started/locally/
+    { version = "^2.0.1", markers = "sys_platform == 'linux'", source = "pytorch-cpu-src" },
+]
```

Comment:
Adding the nxontology-ml dependency as well as torch explicitly for the `Repository "pytorch-cpu-src" does not exist` issue increased installation time on GH actions from 45 seconds to 90 seconds.